### PR TITLE
Update samples

### PR DESCRIPTION
**Pull request by dasuni-30**

### CODE REVIEW

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@App:name("ConnectSiddhiAppsSink")
```
Comment (niveathika) [inline samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi]: Rename as ConnectSiddhiAppsSink1 and ConnectSiddhiAppsSink2

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        c) In the name and amount fields, enter the following and then click Send to send the event.
+            name: chocolate cake
+            amount: 50.50
+        d) Send some more events.
+
+Viewing the Results:
+    INFO {io.siddhi.core.stream.output.sink.LogSink} - ConnectSiddhiAppsSource : OutputStream : Event{timestamp=1569997627918, data=[chocolate cake, 50.5], isExpired=false}
+
+Note:
+    Both "ConnectSiddhiAppsSink" app and "ConnectSiddhiAppsSource" app should be running at the same time in order to view the results on the console.
```
Comment (niveathika) [inline samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi]: Move this as testing sample step 1

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSource.siddhi [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Prerequisites:
+    1) Save this sample
+
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'.
+    2) If the Siddhi application starts successfully, the following messages would be shown on the console.
+        * ConnectSiddhiAppsSource.siddhi - Started Successfully!
+
+Note:
+    Both "ConnectSiddhiAppsSink" app and "ConnectSiddhiAppsSource" app should be running at the same time in order to view the results on the console.
```
Comment (niveathika) [inline samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSource.siddhi]: Move note as executing sample step 1 and say that the other siddhi app has intuctions

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/README.txt [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Purpose:
```
Comment (niveathika) [inline samples/artifacts/DataPipelining/ConnectSiddhiApps/README.txt]: Remove README

File: samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi [hunk 5]
```diff
@@ -25,29 +24,40 @@ Testing the Sample:
...
+define stream IntermediateStream (name string, amount double);
+
+@sink(type='log')
 define stream OutputStream (name string, amount double);
 
-@info(name='EventPassThrough')
+@info(name='EventPublishThrough')
 from SweetProductionStream
 select *
+insert into IntermediateStream;
```
Comment (niveathika) [inline samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi]: Whats the use of Intermediate Stream?  Add the sink to the OutputStream.

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi (hunk 1 above)
Comment (dasuni-30) [reply samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi]: The requested changes are addressed by the commit 6d13673.

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi (hunk 2 above)
Comment (dasuni-30) [reply samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSink.siddhi]: The requested changes are addressed by the commit 6d13673.

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSource.siddhi (hunk 3 above)
Comment (dasuni-30) [reply samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSource.siddhi]: The requested changes are addressed by the commit 6d13673.

File: samples/artifacts/DataPipelining/ConnectSiddhiApps/README.txt (hunk 4 above)
Comment (dasuni-30) [reply samples/artifacts/DataPipelining/ConnectSiddhiApps/README.txt]: The requested changes are addressed by the commit 6d13673.

File: samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi (hunk 5 above)
Comment (dasuni-30) [reply samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi]: The requested changes are addressed by the commit 6d13673.

File: samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi [hunk 6]
```diff
@@ -25,29 +24,36 @@ Testing the Sample:
...
                 * Siddhi App Name  : PublishHttpInXmlFormat
                 * Stream Name      : SweetProductionStream
-        c) In the name and amount fields, enter 'toffees' and '75.6' respectively and then click Send to send the event. 
+        c) In the name and amount fields, enter 'toffees' and '75.6' respectively and then click Send to send the event.
         d) Send more events as desired.
 
 Viewing the Results:
-    See the output on the terminal.
+    See the output on the terminals.
+        Siddhi Console:
```
Comment (niveathika) [inline samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi]: No need for Siddhi console log lets remove and change sample as in the comment,

File: samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi [hunk 7]
```diff
@@ -25,29 +24,36 @@ Testing the Sample:
...
 	@map(type = 'xml'))
+define stream SweetProductionStream (name string, amount double);
+
+@sink(type='log')
 define stream OutputStream (name string, amount double);
 
-@info(name='EventPassThrough')
+@info(name='EventPublishThrough')
 from SweetProductionStream
 select *
```
Comment (niveathika) [inline samples/artifacts/EventPublishing/PublishHttpInXmlFormat.siddhi]:
```suggestion

define stream SweetProductionStream (name string, amount double);

@sink(type = 'http',
	publisher.url = 'http://localhost:8080/abc',		publisher.url = 'http://localhost:8080/abc',
	@map(type = 'xml'))		@map(type = 'xml'))
define stream OutputStream (name string, amount double);	define stream OutputStream (name string, amount double);

@info(name='EventPassThrough')	
from SweetProductionStream
select  * 
insert into OutputStream```

**Pull request merged**